react-frontend-app/src/App.js

```diff
@@ -20,25 +20,61 @@ import BarGraph from './components/BarGraph'
 function App() {
   const [surname, setSurname] = useState('');
   const [zipcode, setZip] = useState('');
-  var prediction = {};
+
+  // ZRP inputs
+  const [firstName, setFirst] = useState('');
```

Comment:
Moving forward, we'll also want to change these names to be more unique/descriptive to Surgeo and the ZRP.  Could you change names like cong and distinguish between surgeo zipcode/surname and zrp zipcode/surname?

Reply:
Building off of that, this is a bit of a design choice, but I think it would be cleaner to also refactor by grouping up some of the states. The reference below might be useful:

https://stackoverflow.com/questions/59813926/usestate-to-update-multiple-values-in-react

Reply:
Clarification: My bad, just thought that we should rename things like cong, and then for readability it would be good to refactor it to be more clear.

Reply:
@dzheng21 i'm not sure if we need to distinguish between surgeo/ZRP surname/zipcode. I've renamed them all to reflect the ZRP inputs that Rakesh decided to use because they're a bit more comprehensive